lib/App/git/ship/perl.pm

```diff
@@ -139,7 +139,7 @@ sub ship {
   $self->system(qw( git add Makefile.PL ), $changelog, $self->_filename('readme'));
   $self->system(qw( git commit -a -m ), $self->_changes_to_commit_message);
   $self->SUPER::ship(@_);    # after all the changes
-  $uploader->upload_file($dist_file);
+  $uploader->upload_file($dist_file) unless $ENV{GIT_SHIP_NO_UPLOAD};
```

Comment:
Maybe we should turn $uploader into an attribute and then have:

    $self->uploader->upload_file($dist_file) unless $ENV{GIT_SHIP_UPLOAD} // 1;